app/views/records/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="card mb-3">
+  <%= form_with(model: record, local: true) do |form| %>
```

Comment:
`record` is used here but all others call is `@record`

Reply:
I meant to use locals everywhere but I'm changing accordingly to the `before_action` approach.